harvester_e2e_tests/apis/test_images.py

```diff
@@ -137,6 +158,31 @@ def test_create_with_reuse_display_name(self, api_client, unique_name, fake_imag
 
         _ = api_client.images.delete(unique_name)
 
+    @pytest.mark.dependency(name=["upload_image"])
+    def test_upload_file(self, api_client, unique_name, file_name, size):
```

Comment:
where are fixtures `file_name` and `size`?

Reply:
The `file_name` and `size` value did not set as fixture since they were sent as parameter value via test function call.